docs/banzai/index.rst

```diff
@@ -2,80 +2,136 @@
+This is similar to Source Extractor, but is written purely in Python and Cython. This allows more
+customization.
+
+We estimate the background by taking a 3x3 median filter of the image and the doing a 32x32 block
+average of the image.
+
+We use the default match filter for source detection that is provided by SEP.
+
+We do aperture photometry using an elliptical aperture that is set by 2.5 times the Kron radius. This
+produces approximately the same results as ``FLUX_AUTO`` from SExtractor.
+
+We set the source detection limit at 3 times the global rms of the image. ``MINAREA`` is set to 5,
+the default. This should minimize false detections, but may miss the faintest sources.
+
+The catalog is returned as the ``CAT`` as fits binary table extension of the final science image. The catalog
+has the following columns: the position in pixel coordinates, (``X``, ``Y``), the flux (``Flux``),
+the error in the flux (``Fluxerr``), the semi-major and semi-minor axes (``a``, ``b``),
+and the position angle (``theta``).
+
+
+Astrometry
+==========
+The WCS is found by using Astrometry.net (Lang et al. 2009, arXiv:0910.2233). We use the catalog from
+the source detection (the previous step) as input.
+
+We adopt a code tolerance of 0.003 (a factor of 3 smaller than the default), but increase the centroid
+uncertainty to be 20 pixels. The large centroid uncertainty allows the algorithm to find quads even
+if the initial guess is quite poor and even if there is significant distortion. However, decreasing
+the code tolerance forces the algorithm to only use high quality quads, making the solution more
+robust. We also go deeper into the catalogs (200 quads deep) to increase the chances of a successful
+astrometry solution.
+
+Currently no non-linear distortion is included in the WCS (the current WCS solution only has a center,
+a pixel scale, and a rotation). At worst (in the image corners), the offset between
+coordinates with non-linear distortion terms included and those without are ~5 arcseconds.
+
+Eventually, astrometry.net may be replaced with a purely Python alternative
+(e.g. AliPy; http://obswww.unige.ch/~tewes/alipy/).
+
+
+Master Calibration Frames
+-------------------------
+BANZAI also contains routines to create the master bias, dark and flat frames required for the reduction of
+science frames.  Before we describe how these are created, we introduce an important statistical metric used
+throughout the BANZAI pipeline.
+
+Sigma clipping is a standard technique to reject bad pixels. However, outliers artificially increase the standard
+deviation (std) of the points (which makes outliers appear to be fewer sigma away from the peak) making it difficult to
+produce robust results. We have adopted a slightly different method. We use the median absolute deviation (mad) to
+estimate the scatter of the distribution (traditionally characterized by the standard deviation). The mad is related to the
+std by
+
+\sigma = 1.4826 * mad
```

Comment:
This should be an approximate equal sign.

Reply:
Fixed